terraform/cloud-platform-components/resources/cluster-backup-checker.yaml

```diff
@@ -0,0 +1,23 @@
+--- 
+apiVersion: batch/v1beta1
+aws: 
+  role: 
+kind: CronJob
+metadata: 
+  name: cluster-snapshot-checker-cronjob
+  namespace: monitoring
+spec: 
+  jobTemplate: 
+    spec: 
+      template: 
+        spec: 
+          containers: 
+            - 
+              env: 
+                - 
+                  name: AWS_ACCESS_KEY_ID
+                  value: access_key
+              image: "754256621582.dkr.ecr.eu-west-2.amazonaws.com/cp-team/cp-poornima-dev-module:cluster-backup-checker-7.0"
```

Comment:
The name of this application `cp-poornima-dv-module` will need to be something fit for master.

Reply:
Unwanted file commit removed.